pages/index.js

```diff
@@ -47,16 +66,17 @@ export default function Home() {
               </p>
             </div>
             <div>
-              <h2>License Grant</h2>
+              <h2>Radio Grant</h2>
               <p>
-                If you pass the test by May 31st, we will cover your exam and
-                license fees!
+                If you pass the test by May 31st, we will cover your first
+                radio! Note that Hack Club will not cover the $5 exam fee.
```

Comment:
Copytext fix:

`If you pass your test by May 31st, we'll send you a ham radio! (Note: this is for people under Middle/High Schoolers only!)`


_We don't need the part about the exam fee._

Reply:
Then I think we're being a bit misleading.

Reply:
Nowhere else do we say we cover the exam fee.

Reply:
We should let people know that this isn't free

Reply:
I still firmly think this isn’t needed.

Reply:
Fixed